src/python/espressomd/electrostatic_extensions.pyx

```diff
@@ -27,7 +27,13 @@ from espressomd.utils cimport handle_errors
 
 IF ELECTROSTATICS and P3M:
     cdef class ElectrostaticExtensions(actors.Actor):
-        pass
+        def _check_for_electroneutrality(self):
```

Comment:
please autopep8 this file

Reply:
autopep8 will be applied with the 4.0 release